test/dynamo/test_aot_autograd.py

```diff
@@ -720,6 +722,76 @@ def fn(x):
         opt_fn = torch._dynamo.optimize("aot_eager")(fn)
         self.assertTrue(torch._dynamo.testing.same(fn(x), opt_fn(x)))
 
+    def test_aot_sequence_nr(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.conv1 = torch.nn.Conv2d(
+                    in_channels=16,
+                    out_channels=16,
+                    kernel_size=(1, 1),
+                    stride=1,
+                    padding="same",
+                    bias=True,
+                )
+                self.bn1 = torch.nn.BatchNorm2d(num_features=16)
+                self.relu1 = torch.nn.ReLU()
+                self.fc1 = torch.nn.Linear(in_features=1638400, out_features=1)
+                self.loss_fn = torch.nn.L1Loss()
+
+            def forward(self, x, target):
+                y = x
+                x = self.conv1(x)
+                x = self.bn1(x)
+                x = self.relu1(x)
+                x = x + y
+                x = torch.flatten(x)
+                x = self.fc1(x)
+                output = self.loss_fn(x, target)
+
+                return (output,)
+
+        mod = Model()
+        mod.train()
+        x = torch.rand(100, 16, 32, 32, requires_grad=True)
+        target = torch.rand(1)
+
+        # Use dynamo export to get the fx graph module
+        g_mod, _ = torch._dynamo.export(mod, x, target)
+
+        # aot_export requires a graph mod input of fwd graph
+        # returns the full fwd/bwd graph in graph mod format
+        with torch.enable_grad(), fx_traceback.preserve_node_meta():
+            fx_g, signature = aot_export_module(
+                g_mod,
+                [x, target],
+                trace_joint=True,
+                output_loss_index=0,
+                skip_flatten_joint=True,
+            )
```

Comment:
Could you add to the PR description what the output of this is, i.e. the joint graph + seq_nr information?

Reply:
I just updated the PR description, is that sufficient?

Reply:
Thanks for the additional context there- I was also thinking we could have in the description a example of a traced joint graph, with all the nodes in the forward and backward as well as what the seq_nr in their metas are.

Reply:
Yes - I just added it to the test, thanks for showing me how.